src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="section">
+      <ul class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
It seems like you're setting up modifiers for the `calendar` block. According to BEM methodology, modifiers should be used with the element or block they modify, not as standalone classes. It should be `calendar calendar--start-day-sun calendar--month-length-31`.